app/controllers/offers_controller.rb

```diff
@@ -51,8 +53,10 @@ def confirm_cancel
   # GET /offers/1/accept
   # should only by accessable be either requestor or admin
   def accept
+
     @offer = Offer.find(params[:id])
-    if @offer.request.status == 'open' && @offer.status == 'pending' && @offer.roster.start > DateTime.now()
+    # if @offer.request.status == 'open' && @offer.status == 'pending' && @offer.roster.start > DateTime.now()
+       if @offer.request.status == 'open' && @offer.status == 'pending'
```

Comment:
This looks incorrect. Removing the "@offer.roster.start > DateTime.now()" would allow someone to confirm an offer in the past which is not allowed. Please review.